### PR TITLE
Implement Tooltip

### PR DESCRIPTION
**Pull request by abhidesh128**
Implemented tooltip with status of unread messages.

### CODE REVIEW

**Comment by mimbrero** [comment]
@abhidesh128 you have to manually accept my review

File: src/tray-manager.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ export default class TrayManager {
         }
 
         this.tray.setContextMenu(menu);
+        this.tray.setToolTip('WhatsApp Desktop - ' + this.unread + ' unread messages');
```
Comment (abhidesh128) [inline src/tray-manager.ts:65]: this.tray.setToolTip('WhatsApp Desktop - ' + this.unread + ' unread chats');

**Comment by abhidesh128** [comment]
Sorry @mimbrero, but I guess I did not get what you said. Do I have to review the changed file and suggest changes (if any) ?

**Comment by mimbrero** [comment]
No, just accept my suggestion, but no problem, I'll do it.

**Review by mimbrero — APPROVED** [review]

**Comment by mimbrero** [comment]
I cannot try if this is working, so I'll merge this if you can confirm that this is working.

**Comment by mimbrero** [comment]
I cannot test if this is working, so I'll merge this if you confirm.

**Pull request merged**